MapboxCoreNavigation/VisualInstructionFormatter.swift

```diff
@@ -27,4 +27,22 @@ public class VisualInstructionFormatter: NSObject {
             return nil
         }
     }
+    
+    public func strings(leg: RouteLeg?, step: RouteStep?) -> (String?, String?) {
+        if let currentLeg = leg, let destinationName = currentLeg.destination.name, let step = step, step.maneuverType == .arrive {
+            return("You have arrived", destinationName)
```

Comment:
@bsudekum should we localize this or get it from the step’s instruction? (`You have arrived at your destination` sometimes it also appends `, on the right`) /cc @ericrwolfe

Reply:
We should localize this.

Reply:
We might want to use OSRMTextInstructions for this actually. /cc @1ec5